File: app/src/main/java/com/example/kirill/retrofittry/Interfaces/GetTests.java
package com.example.kirill.retrofittry.Interfaces;

import com.example.kirill.retrofittry.Parsers.Test;
import com.example.kirill.retrofittry.Parsers.Tests;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface GetTests {
    @GET("tests")
    Call<Tests> all();

    @GET("tests/{id}")
    Call<Test> select(@Path("id") int id);
}

File: app/src/main/java/com/example/kirill/retrofitTry/interfaces/GetTests.kt
package com.example.kirill.retrofitTry.interfaces

import com.example.kirill.retrofitTry.Parsers.Test
import com.example.kirill.retrofitTry.Parsers.Tests
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface GetTests {
    @GET("tests")
    fun all(): Call<Tests>

    @GET("tests/{id}")
    fun select(@Path("id") id: Int): Call<Test>
}